tiler-visor/start.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -e
+sed -i -e 's/demo.tegola.io/'$TILER_SERVER_HOST:$TILER_SERVER_PORT'/g' /usr/share/nginx/html/config.json
+sed -i -e 's/localhost:9090/'$TILER_SERVER_HOST:$TILER_SERVER_PORT'/g' /usr/share/nginx/html/capabilities/osm.json
+sed -i -e 's,https://osm.tegola.io,'${http://$TILER_VISOR_HOST:$TILER_VISOR_PORT}',g' /usr/share/nginx/html/config.json
```

Comment:
@Rub21 at some point we will want these to be `https` - can we make `$TILER_VISOR_PROTOCOL` another parameter that will define `http` or `https` ?